monitoring/cluster-monitoring/examining-cluster-metrics.adoc

```diff
@@ -0,0 +1,17 @@
+[id="examining-cluster-metrics"]
+= Examining cluster metrics
+include::modules/common-attributes.adoc[]
+:context: querying-metrics
+
+toc::[]
+
+{product-title} 4 provides a Web interface to Prometheus, which enables you to run Prometheus Query Language (PromQL) queries and examine the metrics visualized on a plot. This provides an extensive overview of the cluster state and enables you to troubleshoot problems.
+
+include::modules/monitoring-contents-of-the-metrics-ui.adoc[leveloffset=+1]
+include::modules/monitoring-running-metrics-queries.adoc[leveloffset=+1]
+include::modules/monitoring-exploring-the-visualized-metrics.adoc[leveloffset=+1]
+
+.Next steps
+
+xref:../../monitoring/cluster-monitoring/prometheus-alertmanager-and-grafana.adoc#prometheus-alertmanager-and-grafana[Access the Prometheus, Alertmanager, and Grafana.]
```

Comment:
_Access the Prometheus, Alertmanager, and Grafana._ interfaces? pages? Besides being grouped together, I'm not sure what the relationship is between these?

Reply:
I was asked (twice) to leave "interfaces" out of this wording due to some technicality (frankly I don't remember what technicality).
These three things is why have the monitoring stack at all. They are how you actually use the stack once you've configured it.